TagV3.0_U575VGT/Core/Src/Sensor Src/LightSensor.c

```diff
@@ -183,3 +179,34 @@ HAL_StatusTypeDef LightSensor_get_part_id(LightSensorHandleTypedef *light_sensor
 HAL_StatusTypeDef LightSensor_get_manufacturer(LightSensorHandleTypedef *light_sensor, ALSManufacIDRegister *dst){
     return HAL_I2C_Mem_Read(light_sensor->i2c_handler, ALS_ADDR << 1, ALS_MANUFAC_ID_ADDR, I2C_MEMADD_SIZE_8BIT, (uint8_t*)dst, sizeof(ALSManufacIDRegister), 100);
 }
+
+
+void priv_lightSensor_sample_timer_expiration(ULONG light_sensor_raw_addr){
```

Comment:
if we want true "private" usage of this function, should we add "static" and leave it as only defined in the C file? 

Not a big deal (or even a problem) at all but more-so styling preference, so I'll let you decide what makes sense to you